src/data/arena-tag.ts

```diff
@@ -926,7 +918,7 @@ class SpikesTag extends DamagingTrapTag {
  */
 class StealthRockTag extends DamagingTrapTag {
   public readonly tagType = ArenaTagType.STEALTH_ROCK;
-  public override get maxLayers() {
+  override get maxLayers() {
```

Comment:
Not sure why `public` got removed.
```suggestion
  public override get maxLayers() {
```

Reply:
Probably a merge resolution.